src/test/kotlin/uk/gov/justice/hmpps/offendersearch/services/MatchServiceTest.kt

```diff
@@ -157,15 +154,19 @@ internal class MatchServiceTest {
   }
 
   @Test
+  @MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
I don't remember having to do this before, the "whenever" below is overriding the one in setup which therefore never gets called, odd